pnpm-lock.yaml

```diff
@@ -18,9 +18,6 @@ importers:
       prettier:
         specifier: ^3.2.5
         version: 3.2.5
-      rollup:
```

Comment:
Not really worth it for this package of course, the `node_modules` overhead is too big here compared to "just" Rollup or equivalent, esp. considering there isn't even JS/TS transforms.